tailwind.config.js

```diff
@@ -10,9 +10,8 @@ module.exports = {
       colors: {
         'zinc-purple': '#332658',
         'zinc-dark-purple': '#1F1735',
-        'zinc-hot-pink': '#F43EF3',
```

Comment:
remove 'zinc-hot-pink': '#F43EF3' as it is not WCAG compliant w/ the background colors https://github.com/zinc-collective/www.zinc.coop/pull/116#issuecomment-1594055940